examples/example_ecostress.py

```diff
@@ -0,0 +1,70 @@
+#Copyright (C) 2017-2018 COAL Developers
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth
+# Floor, Boston, MA 02110-1301, USA.
+# encoding: utf-8
+
+'''
+example_ecostress.py -- a script which will generate envi .sli and .hdr convolved library
+files of `ECOStress Spectral Library <https://speclib.jpl.nasa.gov/>`_
+
+Dependencies
+`ECOStress Spectral Library <https://speclib.jpl.nasa.gov/>`_
```

Comment:
I think we should be more descriptive here. 
The [downloads page](https://speclib.jpl.nasa.gov/download) includes the following artifacts
```
All Lunar (17 files)
All Man-made (72 files)
All Meteorites (59 files)
All Minerals (3104 files)
All Non-Photosynthetic Vegetation (62 files)
All Rock (647 files)
All Soil (120 files)
All Vegetation (1031 files)
All Water (9 files)

Or

ALL Spectra (5104 files)
```
Which of the above do we download?

Reply:
ALL Spectra, I have now made a comment in the code to specify